lib/lexus-spec.js

```diff
@@ -21,7 +21,7 @@ function bundle(vsn) {
   });
 }
 
-function locate(vsn) {
+function locate() {
```

Comment:
We can actually drop the `vsn` argument from all functions in here... it's not actually used anywhere anymore.